apps/website/src/server/scheduler.ts

```diff
@@ -6,6 +6,8 @@ import { prisma } from "@/server/db/client";
 
 type TaskConfig = ScheduledTasksConfig["tasks"][number];
 
+const MAX_EXECUTION_DURATION = 1000 * 60 * 5; // 5 minutes
```

Comment:
Should this be something defined on each task, in case we have a known-long-running task?

Reply:
We could do that yeah. Not sure if its worth though. I chose 5m/300s because thats the limit for vercel functions in the Pro plan. So it is kinda the maximum it could ever run. I also do not expect for tasks to fail like this anymore, so I see it as a failsafe.

Reply:
Forgot to link the docs: https://vercel.com/docs/functions/limitations

Reply:
Ah cool, that's fine then 👍